packages/docusaurus-theme-common/src/hooks/useWindowSize.ts

```diff
@@ -43,17 +46,17 @@ const DevSimulateSSR = process.env.NODE_ENV === 'development' && true;
  * In development mode, this hook will still return `"ssr"` for one second, to
  * catch potential layout shifts, similar to strict mode calling effects twice.
  */
-export function useWindowSize(): WindowSize {
+export function useWindowSize(desktopThresholdWidth = 996): WindowSize {
```

Comment:
Not fan of using a single arg there, please use a flexible object instead because we don't want to do a breaking change if we later introduce a new breakpoint some day or if you or someone else come with the need for 3 different sizes. If we add flexibility for your use case, let's make it flexible enough for other use cases as well.

This lib has a decent API to copy imho: https://github.com/iiroj/use-breakpoint

```js
const BREAKPOINTS = { mobile: 0, tablet: 768, desktop: 1280 }

const CurrentBreakpoint = () => {
  const { breakpoint, maxWidth, minWidth } = useBreakpoint(
    BREAKPOINTS,
    'desktop'
  )
  return <p>The current breakpoint is {breakpoint}!</p>
}
```

(by the way you could as well use this lib directly in your code, similar behavior to what we implemented here).